oracle/blockoracle_test.go

```diff
@@ -288,3 +289,14 @@ func newBlockWithEligibility(layerID types.LayerID, nodeID types.NodeId, atxID t
 	db.atxs[ed.PublicKey().String()][layerID] = atxID
 	return block
 }
+
+func TestBlockEligibility_calc(t *testing.T) {
+	r := require.New(t)
+	atxH := types.NewActivationTxWithChallenge(types.NIPSTChallenge{PubLayerIdx: 0}, address.Address{}, 1, nil, nil)
+	atxH.ActiveSetSize = 10
+	atxDb := &mockAtxDB{atxH: &atxH.ActivationTxHeader}
+	genSetSize := uint32(0)
+	o := NewMinerBlockOracle(10, genSetSize, 1, atxDb, &EpochBeaconProvider{}, vrfSigner, nodeID, func() bool { return true }, log.NewDefault(t.Name()))
+	err := o.calcEligibilityProofs(1)
+	r.Error(err) // a hack to make sure we got genesis active set size on genesis
```

Comment:
Please replace with `r.EqualError`. If this breaks, it would be very difficult to understand which error we're expecting.

Reply:
Fixed